### PR TITLE
Make VTests fail when mscore crashes during PNG generation

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #9777 

I'm also working on (temporary) fixes for the crashes.

### CODE REVIEW

**Comment by cbjeukendrup** [comment]
Yay, the test is faling as expected. So now I'll push the commits that "fix" the crashes. Note those fixes will only be effective when this PR is merged, because the VTests check also builds the master branch, which doesn't have the fixes yet.

@Nick-Mazuk I think the second of the two fixes is something for you to look at.

**Review by Nick-Mazuk — COMMENTED** [review]
So the end goal of this code is to get the chords that the beam attaches to.

Does MuseScore support beams that hang over rests at the endpoints?

File: thirdparty/haw_logger/logger/log_base.h [hunk 1]
```diff
@@ -42,7 +42,7 @@
 #define DEPRECATED LOGD() << "This function deprecated!!"
 #define DEPRECATED_USE(use) LOGD() << "This function deprecated!! Use:" << use
 #define NOT_IMPLEMENTED LOGW() << "Not implemented!!"
-#define NOT_IMPL_RETURN NOT_IMPLEMENTED return
+#define NOT_IMPL_RETURN NOT_IMPLEMENTED; return
```
Comment (Jojo-Schmitz) [inline thirdparty/haw_logger/logger/log_base.h:45]: Why?

**Comment by Jojo-Schmitz** [comment]
> Does MuseScore support beams that hang over rests at the endpoints?

Yes

File: thirdparty/haw_logger/logger/log_base.h (hunk 1 above)
Comment (cbjeukendrup) [reply thirdparty/haw_logger/logger/log_base.h:45]: Otherwise it would expand to `LOGW() << "Not implemented!!" return`, wouldn't it? That is not valid.

Comment (Jojo-Schmitz) [reply thirdparty/haw_logger/logger/log_base.h:45]: Oops, of course. I misses that it is 3 'words'

File: thirdparty/haw_logger/logger/log_base.h [hunk 2]
```diff
@@ -42,7 +42,7 @@
 #define DEPRECATED LOGD() << "This function deprecated!!"
```
Comment (Jojo-Schmitz) [inline thirdparty/haw_logger/logger/log_base.h:42]: Here an "is" is missing. And "function" probably should be "method"?

File: thirdparty/haw_logger/logger/log_base.h [hunk 3]
```diff
@@ -42,7 +42,7 @@
...
 #define DEPRECATED_USE(use) LOGD() << "This function deprecated!! Use:" << use
```
Comment (Jojo-Schmitz) [inline thirdparty/haw_logger/logger/log_base.h:43]: same

Comment (cbjeukendrup) [reply thirdparty/haw_logger/logger/log_base.h:43]: I think I'll rather remove them altogether. They are not used, as far as I can find.

Comment (Jojo-Schmitz) [reply thirdparty/haw_logger/logger/log_base.h:43]: They are taken from the original source. Maybe getting them fixed there is in order, @igorkorsukov ?

Comment (igorkorsukov) [reply thirdparty/haw_logger/logger/log_base.h:43]: Thanks.
I will fix them later

**Pull request merged**